src/aria/jsunit/TestWrapper.js

```diff
@@ -51,6 +62,41 @@ module.exports = Aria.classDefinition({
         },
 
         /**
+         * Finds SCRIPT nodes of the top-level document that have specific `data-extrascript` attribute
+         * and injects them into an iframe. If there are no such scripts, it does nothing.
+         * It calls the callback
+         * @param {Document} subDocument document element of the iframe
+         * @param {String} whereToInsert expected value of `data-extrascript` attribute
+         * @param {aria.core.CfgBeans.Callback} callback
+         */
+        _injectExtraScriptsIntoFrame : function (subWindow, whereToInsert, callback) {
+            var document = Aria.$window.document;
+            var subDocument = subWindow.document;
+            var scripts = document.querySelectorAll('script') || document.scripts;
```

Comment:
Also, rather than using `querySelectorAll` which is generic for all CSS selectors, you should probably use `getElementsByTagName`.